onecore_maintenance_extension/views/maintenance_views.xml

```diff
@@ -1,6 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+        <!-- This extends the search bar and what fields are searchable -->
```

Comment:
👍

Reply:
Tack! Drog tillbaka den till "in progress" då jag ska lägga till ett par fält på samma sätt. Men mergar when done dårååå 🙏